src/content/nav_tabs.html

```diff
@@ -236,45 +240,45 @@
 	<div class="col-md-12">
 		<h3>Nav Tabs Grid</h3>
 
-		<blockquote class="blockquote-sm blockquote-success">
+		<blockquote class="blockquote blockquote-sm blockquote-success">
 			<p>Use bootstrap's grid inside list items in nav-tabs. It doesn't collapse in mobile.</p>
 		</blockquote>
 
 		<ul class="nav nav-tabs" role="tablist">
-			<li class="active col-xs-3" role="presentation"><a aria-controls="fields" href="#fields03" data-toggle="tab" role="tab">Fields</a></li>
-			<li class="col-xs-3" role="presentation"><a aria-controls="settings" href="#settings03" data-toggle="tab" role="tab">Settings</a></li>
-			<li class="disabled col-xs-3" role="presentation"><a href="#documents">Documents</a></li>
-			<li class="dropdown col-xs-3" role="presentation">
-				<a class="dropdown-toggle" data-toggle="dropdown" href="#configuration">Configuration <span class="caret"></span></a>
-				<ul class="dropdown-menu">
-					<li><a href="#1">Setup</a></li>
-					<li><a href="#1">Supported Clients</a></li>
-					<li><a href="#1">Permissions</a></li>
-					<li><a href="#1">Sharing</a></li>
-					<li class="divider"></li>
-					<li><a href="#1">Scope</a></li>
-				</ul>
+			<li class="nav-item col-xs-3" role="presentation"><a class="nav-link active" aria-controls="fields" href="#fields04" data-toggle="tab" role="tab">Fields</a></li>
+			<li class="nav-item col-xs-3" role="presentation"><a class="nav-link" aria-controls="settings" href="#settings04" data-toggle="tab" role="tab">Settings</a></li>
+			<li class="nav-item col-xs-3" role="presentation"><a class="nav-link disabled" href="#documents">Documents</a></li>
+			<li class="dropdown nav-item col-xs-3 row" role="presentation">
```

Comment:
For some reason the ```Configuration``` tab would not align with the other tabs unless I added the ```row``` class. There may be a better way to fix this.